lib/interactive.dart

```diff
@@ -194,9 +198,53 @@ class Interactive {
       peopleByLabel[key].add(people);
     });
 
+    for (var i = 0; i < xAxisCategories.length; ++i) {
+      var colSVG = svg.SvgElement.tag("g");
+
+      var colData = peopleByLabel[xAxisCategories[i].value];
+      num colOffset = (i + 0.5) * (chartWidth / xAxisCategories.length);
```

Comment:
There's a lot of numbers floating around here - consider colOffsetPx

Reply:
Fixed